### ISSUE TITLE
How to load custom data

### ISSUE THREAD

**Issue by navjeetc** [Senzing/senzing-up#38]
**What can be improved?**

Steps to be executed to load custom data

**Additional context**

Assuming the custom data files (.csv) have been placed in _senzing-up-demonstration/g2/python/demo/sample_ dir, what commands need to be executed to load data from these csv files.

**Comment by navjeetc** [comment]
@docktermj Can you provide the steps, until docs are updated? I am stuck because of this.

**Comment by docktermj** [comment]
Hi @navjeetc   Here you go...

Copy your `csv` file to `~/senzing-up-demonstration/var` so that it can be see inside the docker container.  Example:

```console
cp /opt/senzing/g2/python/demo/sample/sample_person.csv ~/senzing-up-demonstration/var/sample_person.csv
```

Make sure docker network is up.  Example:

```console
sudo docker network create senzing-up
```

Bring up the Senzing console docker image.  Example:

```console

~/senzing-up-demonstration/docker-bin/senzing-console.sh
```

In the Senzing console,  verify your file is there. Example:

```console
ls -la /var/opt/senzing
```

In the Senzing console, use `G2Loader.py` to load the data. Example

```
G2Loader.py  -f /var/opt/senzing/sample_person.csv/?data_source=CUSTOMERS,file_format=CSV
``

For more information on `G2Loader.py`, see https://senzing.zendesk.com/hc/en-us/articles/115004450368-Getting-Started

If more information is needed, let me know.

**Comment by docktermj** [comment]
Added https://github.com/Senzing/senzing-up/blob/master/docs/tutorial.md#demonstration-6

**Comment by docktermj** [comment]
@navjeetc Let me know if the documentation at https://github.com/Senzing/senzing-up/blob/master/docs/tutorial.md#demonstration-6 suffices.

**Comment by navjeetc** [comment]
Documentation looks good. Looks like the loading process is very particular about memory needed. I had to do multiple attempts before I succeeded. It might be worth mentioning the use of -T (test mode but does not load data) and also the difference between -p and -f options (or point docs that explain these options). Also if one is using their own datasource (which is the case for my data), then it is good to know that one can use _G2ConfigTool.py_ and add data sources using command
`addDataSource {"dataSource": 'MY_DATA_SOURCE'}`
